src/utils/config.js

```diff
@@ -142,6 +142,9 @@ const DEFAULTS = {
     ariaLive: 'polite',
     ariaAtomic: 'true',
     role: null
+  },
+  vBTooltip: {
```

Comment:
Change this to `BTooltip` so that it will apply to both the directive version and the component versions.